h264/parse.go

```diff
@@ -108,7 +108,7 @@ func readMe(r bitio.Reader, chromaArrayType int, mpm macroblockPredictionMode) (
 	case 0, 3:
 		i1 = 1
 	default:
-		return 0, errors.New("invalid chromaArrayType")
+		return 0, errInvalidCAT
```

Comment:
Are these errors every likely to be exposed outside of the package, i.e., to callers?